tests/unit/util/tuple.cpp

```diff
@@ -371,6 +371,21 @@ void tuple_cat_test()
         HPX_TEST_EQ((*hpx::util::get<1>(result)), 1);
         HPX_TEST_EQ((*hpx::util::get<2>(result)), 2);
     }
+
+    // Don't move references unconditionally
```

Comment:
This comment is misleading, the test below doesn't contain any checks for moves being performed or not performed.